spark/src/test/scala/org/apache/spark/sql/delta/deletionvectors/DeletionVectorsSuite.scala

```diff
@@ -693,7 +694,8 @@ class DeletionVectorsSuite extends QueryTest
   }
 
   test("absolute DV path with not-encoded special characters") {
-    withTempDir { dir =>
+    // This test uses hand-crafted path with special characters.
```

Comment:
For the other comment as well
```suggestion
    // Do not test with a prefix that needs URL standard escaping. This test uses hand-crafted path with special characters.
```